tasks/trace_agent.py

```diff
@@ -83,3 +83,28 @@ def integration_tests(ctx, install_deps=False, race=False, remote_docker=False):
 
     for prefix in prefixes:
         ctx.run("{} {}".format(go_cmd, prefix))
+
+@task
+def cross_compile(ctx, tag=""):
+    """
+    Cross-compiles the trace-agent binaries. Use the "--tag=X" argument to specify build tag.
+    """
+    if tag == "":
```

Comment:
You may also consider `if not tag`, but that would cover the cases of `None` and `""` (and `0`, although that seems unlikely here). Sometimes you want to explicitly catch one or the other but not both.